ruby_on_rails/rspec.md

```diff
@@ -21,11 +21,12 @@ group :test do
 end
 ```
 
-You should know exactly why you are adding each one of them, why is necessary
+You should know exactly why you are adding each one of them and why is necessary.
 
 ## Configuration
 
 * Install rspec via `rails generate rspec:install`
+* Create a bin stub ith `bundle binstubs rspec-core`
```

Comment:
typo `with`

Reply:
just to understand ist: this creates also a `bin/rspec`. is there other reasons to do it? https://bundler.io/man/bundle-binstubs.1.html

Reply:
no. that's the main reason

Reply:
by convention we have the `bin/check` or `fastcheck` which does that. But maybe we want to use `bin/rspec` inside them?

Reply:
Yes, I would. I'll update these scripts.